satpy/cf/__init__.py

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
```

Comment:
We've been talking in the core devs lately about how these two lines are no longer necessary. Feel free to remove them from all modules you modified in this PR (or leave them).